src/components/Review.jsx

```diff
@@ -57,7 +58,7 @@ const Review = () => {
                     to={{ pathname: `/review/${article.id}` }}
                     style={{ float: "right" }}
                   >
-                    <Button size="tiny" id={"checkout-article-" + article.id}>
+                    <Button color={color} size="tiny" id={"checkout-article-" + article.id}>
```

Comment:
We change the color of the currently previewed article checkout button, but the others are not disabled, they just look like they are?

Reply:
They are not disabled, but the one you are currently previewing is highlighted, so it's easier to find in the list.